src/services/logger.js

```diff
@@ -23,14 +23,20 @@ class Logger {
       color: null,
       prefix: null,
       std: null,
+      lineColor: null,
       ...options,
     };
 
     let actualPrefix = '';
     if ([undefined, null, ''].indexOf(options.prefix) === -1) actualPrefix = `${options.prefix} `;
     if (actualPrefix && options.color) actualPrefix = chalk.bold[options.color](actualPrefix);
 
-    const actualMessage = `${actualPrefix}${message} \n`;
+    let actualMessage;
+    if (options.lineColor) {
+      actualMessage = `${actualPrefix}${chalk[options.lineColor](message)} \n`;
+    } else {
+      actualMessage = `${actualPrefix}${message} \n`;
+    }
```

Comment:
```suggestion
    let actualMessage = message;
    if (options.lineColor) {
      actualMessage = `${chalk[options.lineColor](message)}`;
    }
    actualMessage = `${actualPrefix}${actualMessage}\n`;
```

Don't know why we let an space before the new line character 🤷‍♂️ Removing it does not seem to impact the test suite.